src/bufr/BufrParser/BufrDescription.cpp

```diff
@@ -21,6 +21,8 @@ namespace
     namespace ConfKeys
     {
         const char* Filename = "obsdatain";
+        const char* IsStdFormat = "isStdFormat";
```

Comment:
suggest isWMOFormat instead of isStdFormat, just to be explicit.

Reply:
curious, what is considered "Std"

Reply:
So the NCEP bufr files we use here have the BUFR table included within the first couple of messges inside these BUFR files (basically the information you get when you do gettab.x or call dxdump). The "standard" WMO BUFR files do not include this information as part of the BUFR file, they get this info from external tables which are loaded seperatly. "STD" is the version of BUFR that follows the official WMO BUFR standard... I guess.